CHANGELOG

```diff
@@ -1,3 +1,24 @@
+Version 0.5.0 (2020-05-05)
+--------------------------
+Loader: replace Scio metrics with Dropwizard metrics (close #82)
```

Comment:
`close`?

Reply:
Our guidelines for commit messages are that they should have an imperative mood. Since this is part of the commit message, it is more consistent this way. (Otherwise, it should be 'Loader: replaces Scio metrics with Dropwizard metrics (closes #82)'.)

It works the same way in terms of automatic closing of issues by Gtihub.

Reply:
I wasn't expecting any `close`, `closes` etc in CHANGELOG

Reply:
🤦 Of course...

Reply:
That's done now @oguzhanunlu. Thanks for spotting it!